.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 os : osx
```

Comment:
we also used brew for installing gradle, it is working well with java8 now after doing brew install, so perhaps we can just keep the osx here?